### ISSUE TITLE
[fastpages] Create RNN model for moving classification

### ISSUE THREAD

**Issue by ShiYang1101** [ShiYang1101/Music_transcription_fastai#14]
Create a model for the MusicNet dataset, which includes rolling classification of instrument and pitch as time goes by. The challenge will be defining a sequence of label for tensorflow keras LSTM model, with return_sequence set to tre. The output label is 2 dimensinal numpy array for each instrument, and each instrument should consists of pitch classification with time bins.
